bundle/manifests/search-v2-operator.clusterserviceversion.yaml

```diff
@@ -194,7 +194,10 @@ spec:
           - create
         serviceAccountName: search-v2-operator
       deployments:
-      - name: search-v2-operator-controller-manager
+      - label:
```

Comment:
**Note to reviewers:** Added label shouldn't be a problem. Also I added the `app=search` label.